charger/ocpp/cp_core.go

```diff
@@ -21,6 +21,14 @@ func (cp *CP) Authorize(request *core.AuthorizeRequest) (*core.AuthorizeConfirma
 		},
 	}
 
+	// Set idTag for all connectors in available state
```

Comment:
Das fühlt sich komisch an. Wenn die WB das Tag bei der Initialisierung mit gibt- warum ist das hier notwendig?

Reply:
Korrigier mich wenn ich falsch liege aber die Logik aus dem init / cp_setup läuft beim Start wenn der erste Kontakt mit der Wallbox aufgenommen wird. Hier werden die Einstellungen abgeglichen und z.B. die Plug and Charge ID übernommen (falls konfiguriert).

Der Auth / cp_core wird ausgeführt wenn eine RFID Karte an die Wallbox gehalten wird um eine neue Transaktion zu autorisieren. Hier können wir also den idTag für den bevorstehenden Ladevorgang abgreifen.

Die Art und Weisse wie man das am besten machen kann ohne den konkreten Connector zu kennen ist wohl die große Frage.